cabinet/cabinet.py

```diff
@@ -56,6 +56,20 @@ def get_all(self):
     def get(self, name):
         return self._vault.get(name)
 
+    def get_by_tags(self, tags=[]):
```

Comment:
No worries, is a very common confusion.
Well, lists are mutable, if you want to use a list-ish data type you can use a tuple. Or do you have something else in mind?